app/frontend/actions.ts

```diff
@@ -280,7 +281,7 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
   }
 
   const logout = () => {
-    wallet = null
+    account = null
```

Comment:
I see that the `wallet` variable is still in use, shouldn't that be set to null as well?

Reply:
Fixed.